rubocop.yml

```diff
@@ -1,33 +1,229 @@
 require:
   - rubocop-performance
 
+inherit_mode:
+  merge:
+    - Exclude
+
 AllCops:
   TargetRubyVersion: 2.6
```

Comment:
Bump target Ruby version to 3.0?

Reply:
@billdias do we have any apps that are not yet 3.0?